js/src/miscellaneous.js

```diff
@@ -83,12 +83,11 @@ module.exports = (common) => {
     })
 
     // must be last test to run
-    it('.stop', (done) => {
+    it('.stop', function (done) {
+      this.timeout(10 * 1000)
       // TODO: go-ipfs returns an error, https://github.com/ipfs/go-ipfs/issues/4078
       ipfs.stop((err) => {
-        if (err && err.message !== 'read ECONNRESET') {
```

Comment:
Why the removal.  Normally `ipfs.stop` should not return an error.  There is a bug in go, in that it closes the connection before sending the result.

Reply:
The reason is that the error message on Browsers is different from `read ECONNRESET`. So this check would fail, and the whole test case would fail. Hence I allow one error (no matter which message) to happen (due to the bug in Go).